mainhandler/kubescapehandler.go

```diff
@@ -232,9 +232,11 @@ func (actionHandler *ActionHandler) kubescapeScan(ctx context.Context) error {
 		scanID:   response.ID,
 	}
 
-	nextHandledTime := time.Duration(WaitTimeForKubescapeScanResponse * time.Second)
-	commandResponseData := createNewCommandResponseData(KubescapeResponse, HandleKubescapeResponse, data, &nextHandledTime)
-	insertNewCommandResponseData(actionHandler.commandResponseChannel, commandResponseData)
+	if actionHandler.eventReceiverRestURL != "" {
```

Comment:
How about we have a function like: `IsServiceDiscoveryEnabled()` instead of  having this (`actionHandler.eventReceiverRestURL != ""`) statement all over the code

Reply:
I'd rather have the mock reporter, because it's not about having service discovery

Reply:
I did it with a private field